OSDOCS-4387: adds zstream update for 4.11.12 (#52260)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

==== Updating

To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.

[id="ocp-4-11-12"]
=== RHSA-2022:7201 - {product-title} 4.11.12 bug fix and security update

Issued: 2022-10-31

> {product-title} release 4.11.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:7201[RHSA-2022:7201] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7200[RHBA-2022:7200] advisory.

Reviewer: ```suggestion
{product-title} release 4.11.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:7201[RHSA-2022:7201] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7200[RHBA-2022:7200] advisory.
```

Author: I don't disagree wit this suggestion. Unfortunately, this is stock text that I can't change unless I change it across all branches' RNs.

Reviewer: Gotcha! No worries, yeah that would be too much for this small nit! 👍

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.11.12 --pullspecs
----

[id="ocp-4-11-12-known-issues"]
==== Known issues

> * When upgrading PTP Operator from 4.10 to 4.11, OpenShift-ptp subscription reports the error: `no channel heads found in channel stable of package ptp-operator`. However, the operator correctly updates. (link:https://issues.redhat.com/browse/OCPBUGS-2042[*OCPBUGS-2042*])

Reviewer: Just a question that I think I got confused at, the operator correctly updates even when the error is displayed?

Author: Yeah, I agree it is confusing. I took it to mean what you are saying as well. I'm checking to make sure that I'm correctly interpreting it.

Reviewer: If thats the case, then that might be worth mentioning in that last sentence. Something like: 
"However, the operator still updates correctly when this error is displayed" or along those lines 🙂

> * If a cluster is incrementally updated from a version less than or equal to 4.9, the `namespace` `openshift-dns` may not contain the required pod-security labels required for future version updates. (link:https://issues.redhat.com/browse/OCPBUGS-1549[*OCPBUGS-1549*])

Reviewer: I think 
```suggestion
* If a cluster is incrementally updated from a version less than or equal to 4.9, the `openshift-dns` `namespace` may not contain the required pod-security labels required for future version updates. (link:https://issues.redhat.com/browse/OCPBUGS-1549[*OCPBUGS-1549*])
```
reads a bit more clearer since `openshift-dns` is the name of the `namespace`. Ignore if thats not the case, that was just how I read it!

[id="ocp-4-11-12-notable-technical-changes"]
==== Notable Technical Changes

> * With this release, when the service account issuer is changed, existing bound service tokens are no longer invalidated immediately. Instead, when the service account issuer is changed, the previous service account issuer continues to be trusted for 24 hours.

Reviewer 2: Would you mind making this one tweak that QE suggested?

```suggestion
* With this release, when the service account issuer is changed to a custom one, existing bound service tokens are no longer invalidated immediately. Instead, when the service account issuer is changed, the previous service account issuer continues to be trusted for 24 hours.
```

> * With this release, when the service account issuer is changed to a custome one, existing bound service tokens are no longer invalidated immediately. Instead, when the service account issuer is changed, the previous service account issuer continues to be trusted for 24 hours.

Reviewer: ```suggestion
* With this release, when the service account issuer is changed to a custom one, and existing bound service tokens are no longer invalidated immediately. Instead, when the service account issuer is changed, the previous service account issuer continues to be trusted for 24 hours.
```
